src/index.html

```diff
@@ -13,20 +13,30 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="/src/style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
+    <form
+      class="search-container"
+      action="#"
+      method="post"
       data-qa="keypress"
```

Comment:
Keep your attributes correctly formatted. Since the `<form>` element has more than two attributes, start each one on a new line with 2-space indentation related to the tag.